### ISSUE TITLE
Insufficient color contrast for text links

### ISSUE THREAD

**Issue by terichadbourne** [ipfs/ipfs-docs#293]
In working on a fork of this site I discovered that the `$accentColor` that creates the text links on the beta site (both in-line and on the sidebar) doesn't pass contrast checks for accessibility for normal size font on any of the three backgrounds I see it on. Just dropping these reports here (from https://webaim.org/resources/contrastchecker/) in case they're helpful for future accessibility reviews: 

![image](https://user-images.githubusercontent.com/19171465/76878781-b236e780-684b-11ea-9ca9-e9917ef74966.png)

![image](https://user-images.githubusercontent.com/19171465/76878829-c1b63080-684b-11ea-8c2e-ecd33b6425a8.png)

![image](https://user-images.githubusercontent.com/19171465/76878991-fe822780-684b-11ea-8734-a0a652ed9e12.png)

**Comment by ericronne** [comment]
The design `#377F85`

![image](https://user-images.githubusercontent.com/9259185/82903645-dc311800-9f26-11ea-86f4-fcd35ab63664.png)

**Comment by jessicaschilling** [comment]
If you use `ipfs-css`, the updated teal is `.teal { color: #378085; }` (see https://github.com/ipfs-shipyard/ipfs-css/blob/master/ipfs.css), which passes AA contrast.

**Comment by johnnymatthews** [comment]
## Tasks

- [ ] Import IPFS-CSS into this repo to replace the current styles.
- [ ] Re-run the above accessibility tests. This site should pass all tests.

**Comment by kstuart** [comment]
@johnnymatthews as indicated `ipfs-css` styles will only pass the AA contrast tests, not the AAA tests, which would require a much darker link colour.  You're requirement should probably be updated to only require AA tests to pass.

**Comment by johnnymatthews** [comment]
Good point @kstuart, thanks for picking up on that. Updated the tasks.

**Comment by denim2x** [comment]
I'm working on it

**Comment by rishabhkeshan** [comment]
Hey @johnnymatthews it's been long there hasn't been an update, can I be assigned the issue?

**Comment by johnnymatthews** [comment]
Absolutely! Thanks for picking this up @rishabhkeshan.

**Comment by rishabhkeshan** [comment]
Hey @johnnymatthews just a question, should I make a new branch and submit a PR or submit a PR to main?

**Comment by rishabhkeshan** [comment]
Hey @johnnymatthews, please have a look.

**Comment by rishabhkeshan** [comment]
@johnnymatthews any update?

**Comment by johnnymatthews** [comment]
Hey @rishabhkeshan, sorry for the delay in getting back to you here. Thanks for the PR! One of our front-end devs will give it a review before it gets merged.

**Comment by rishabhkeshan** [comment]
Hey @johnnymatthews @hsanjuan I have done the final changes, could you review the same.

**Comment by rishabhkeshan** [comment]
Hey @johnnymatthews my PR has been approved.

**Comment by rishabhkeshan** [comment]
Hey @johnnymatthews when will my PR be merged and how will I receive the bounty?

**Comment by rishabhkeshan** [comment]
Hey @johnnymatthews what's the procedure for receiving the bounty

**Comment by johnnymatthews** [comment]
Ah sorry, there's meant to be an automated thing that runs. But looks like it didn't. Drop me an email at johnny@protocol.ai and we'll start the bounty process!
